charts/sourcegraph-migrator/README.md

```diff
@@ -0,0 +1,85 @@
+# Sourcegraph Migrator Helm Chart
+
+This chart contains a single [Job] to run [migrator] operations. It is a supplemental chart for the parent [sourcegraph/sourcegraph] Helm Chart.
+
+Use cases:
+
+- Perform initial migrations against external PostgreSQL databases prior to the Sourcegraph deployment
+- Perform database migrations prior to upgrading the Sourcegraph deployment
+- Troubleshoot a [dirty database]
+
+## Requirements
+
+* [Helm 3 CLI](https://helm.sh/docs/intro/install/)
+* Kubernetes 1.19 or greater
+
+## Installation
+
+Add the Sourcegraph charts repo to Helm:
+
+```sh
+helm repo add sourcegraph https://sourcegraph.github.io/deploy-sourcegraph-helm/
+```
+
+## Usage
+
+> If you are not using external databases, the chart has to be installed in the same namespace as the parent [sourcegraph/sourcegraph] chart
+
+[sourcegraph/sourcegraph-migrator] chart requires the correct `PG*` and `CODEINTEL_PG*` environment variables to be configured at `migrator.env`. Learn more about [using your own PostgreSQL server]. `PG*` and `CODEINTEL_PG*` environment variables are compatible with the `frontend.env` values from the parent [sourcegraph/sourcegraph] chart.
```

Comment:
Code Insights too!

Reply:
> Code Insights too!

It looks like we have more docs to update!

https://docs.sourcegraph.com/admin/external_services/postgres#instructions